src/isPasswordActual.test.js

```diff
@@ -14,15 +14,44 @@ describe(`Function 'isPasswordActual':`, () => {
   });
 
   it(`should return a string`, () => {
+    const { year, month, date: dateNow } = today;
 
+    expect(typeof isPasswordActual(year, month, dateNow))
+      .toBe('string');
   });
 
   it(`should ask to change the password if was changed a year ago`, () => {
-    const lastYear = isPasswordActual(today.year - 1, today.month, today.date);
+    const lastYear
+      = isPasswordActual(today.year - 1, today.month, today.date);
 
     expect(lastYear)
       .toBe('Immediately change the password!');
   });
 
-  // write more tests here
+  it(`should ask to change the password `
+    + `if was changed more than 60 days`, () => {
+    const lastYear
+      = isPasswordActual(today.year, today.month, today.date - 61);
+
+    expect(lastYear)
+      .toBe('Immediately change the password!');
+  });
+
+  it(`should suggest to change the password if was changed`
+      + `more than 30 days but less then 60 days`, () => {
```

Comment:
```suggestion
  it('should suggest to change the password if was changed'
      + 'more than 30 days but less then 60 days', () => {
```
you can use common string literal instead of string template here - FYI